modules/compbox/templates/http-wsgi.cfg.erb

```diff
@@ -8,5 +8,5 @@ app.wsgi_app = DispatcherMiddleware(app.wsgi_app, {
     '/comp-api': app.wsgi_app
 })
 
-bind = '[::]:5112'
+bind = '[::1]:5112'
```

Comment:
Question: Given #5, should this be a v4 loopback?

```suggestion
bind = '127.0.0.1:5112'
```

Reply:
We could, but we would also need to update the nginx config (which uses `http://localhost:5112`) otherwise this may fail to connect on Mythic.